src/cljc/ataru/schema/form_schema.cljc

```diff
@@ -41,6 +41,10 @@
                    :haku-name         s/Str
                    :application-count s/Int})
 
+(s/defschema Haku2 {:haku              s/Str
```

Comment:
But you found it! There is now s/Any and a missing schema for the new interface. And I've forgotten it, so this relic was good.